docs/source/articles/configuration-fields.html.md

```diff
@@ -98,13 +98,13 @@ Of note, `duration` allows you to define a field for periods of time like `3.day
 
 When defining a configuration field, there are a number of options you can provide.
 
-- `allow_blank`
+- `required`
 
-  Defaults to false. Whether or not the field can be set to a blank value by a user. Blank values submitted for a field that does not allow it will be set to the field's default value. The admin UI will also prevent user's from submitting blank field values when this is false.
+  Whether or not the field is required, defaults to `false`. Blank values submitted for a field that does not allow it will be set to the field's default value. The admin UI will also prevent user's from submitting blank field values when this is `false`.
```

Comment:
This set of sentences is really confusing in the context of `requires` over the old `allow_blank`

```suggestion
  Whether or not the field is required, defaults to `false`. The admin UI will prevent user's from submitting blank field values for required fields. If values are submitted in some other way, blank values for a required field will be set to the field's default value.
```